web/src/app/browser/src/hardwareEventKeyboard.ts

```diff
@@ -326,7 +326,7 @@ export default class HardwareEventKeyboard extends HardKeyboard {
             inputEle.form.submit();
           } else {
             const nextElement = this.contextManager.page.findNeighboringInput(inputEle, false);
-            nextElement.focus();
+            nextElement?.focus();
```

Comment:
I wonder why we are not finding any input elements in the page in `findNeighboringInput`? Is there an initialization order issue?

Reply:
Yeah, I wonder that too.